litmus-portal/frontend/src/components/WorkflowCard/CardContent.tsx

```diff
@@ -74,19 +77,23 @@ const CardContent: React.FC<preDefinedWorkflowData> = ({
         onClick={handleClick}
       >
         <div className={classes.cardAnalytics}>
-          {/* {totalRuns ? (
-            <span
-              className={
-                selectedID === workflowID
-                  ? classes.totalRunsSelected
-                  : classes.totalRuns
-              }
-            >
-              {formatCount(totalRuns)}+
+          {chaosinfra === true ? (
+            <span>
+              <Tooltip
+                title="Uses kube-proxy for identification of pod"
```

Comment:
use from translation

Reply:
Will you suggest better way? It's throwing error here
```title = {t('customWorkflowCard.cardTitleInfraChaos')}```

Reply:
Use template strings @oumkale like title=‘${t(a.b)}’